src/unity/python/turicreate/test/test_graph_analytics.py

```diff
@@ -146,6 +146,11 @@ def test_pagerank(self):
             self.assertAlmostEqual(m2.pagerank['pagerank'].sum(), 7087.08, delta=1e-2)
             with self.assertRaises(Exception):
                 assert_frame_equal(m.pagerank.topk('pagerank'), m2.pagerank.topk('pagerank'))
+
+            pr_out = m2['pagerank']
+            with self.assertRaises(Exception):
+                assert_frame_equal(m.pagerank.topk('pagerank'), pr_out.topk('pagerank'))
```

Comment:
Why is this supposed to raise an exception? And what exception specifically?

Reply:
It's suppose to raise an exception because the data frames are different; they're page ranks from different models.

Adding a specific exception types is a good idea. Currently this same line is used several times in this file.